### ISSUE TITLE
Dokan 2.0.0 support

### ISSUE THREAD

**Issue by Liryna** [dokan-dev/dokan-rust#3]
Dokan [2.0.0](https://github.com/dokan-dev/dokany/releases) was just released. There are a few changes in the API [dokan.h](https://github.com/dokan-dev/dokany/compare/v1.5.1.1000...v2.0.0.2000#diff-043a92293d2218cfe605149daa53485589f1f260f6224f7d858ed20e6c5b3e61) and some logic described [here](https://github.com/dokan-dev/dokany/wiki/Update-Dokan-1.1.0-application-to-Dokany-2.0.0) that changed.
It would be great if the wrapper could implement those changes!
And of course, feel free to ping me if any questions or issues arise.

**Comment by DDoSolitary** [comment]
Yeah I noticed the release and will start working on this late this month (when my final exams have finished)

**Comment by Liryna** [comment]
Awesome! Take your time and good with your exams!

**Comment by FirTech** [comment]
I look forward to your support as soon

**Comment by KoltesDigital** [comment]
I just tested the crate, dokan2.dll is installed but indeed dokan1.dll was expected.

@DDoSolitary have you made any progress? I'm not familiar with Dokan, I've just read the docs. If you believe the changes are small between 1 and 2 and you don't have time for that, I may have a try.

**Comment by KoltesDigital** [comment]
I couldn't resist. PR at https://github.com/dokan-dev/dokan-rust/pull/5
